Makefile

```diff
@@ -49,3 +49,13 @@ npm-build:
 update-openfga-model:
 	@printf 'package authorization\n\n// Code generated by Makefile; DO NOT EDIT.\n\nvar AuthModel = `%s`\n' '$(shell fga model transform --file=./internal/authorization/schema.openfga | jq -c)' > ./internal/authorization/auth_model.go
 .PHONY: update-openfga-model
+
+test-e2e:
+	@echo "Running e2e tests"
+	cd ui && npx playwright test
```

Comment:
This should be in the ui makefile and this should call that as in https://github.com/canonical/identity-platform-login-ui/blob/main/Makefile#L45